package.json

```diff
@@ -97,7 +98,7 @@
     "style-loader": "0.23.1",
     "webpack": "4.41.0",
     "webpack-cli": "3.3.9",
-    "webpack-dev-server": "3.2.1",
+    "webpack-dev-server": "3.8.1",
```

Comment:
This line was simply to avoid a local devstack problem with newer webpack-cli not shipping a binary dev-server needed.